manim/tex_font_templates.py

```diff
@@ -0,0 +1,587 @@
+import os
+from .utils.tex import TexTemplateFromFile
+from .mobject.svg.tex_mobject import TextMobject
+from .utils.config_ops import digest_config
+
+
+class TexTemplateFromFontConfig(TexTemplateFromFile):
+    def __init__(self, font, **kwargs):
+        digest_config(self, kwargs)
+        self.font=font
+        if "compiler" in TEX_FONT_CONFIGS[font]:
+            tex_command = TEX_FONT_CONFIGS[font]["compiler"]
+        else:
+            tex_command = "latex"
+        if "output" in TEX_FONT_CONFIGS[font]:
+            tex_output = TEX_FONT_CONFIGS[font]["output"]
+        else:
+            tex_output = ".dvi"
+        self.tex_compiler = {
+            "command": tex_command,
+            "output_format": tex_output,
+        }
+        self.rebuild_cache()
+
+    def rebuild_cache(self):
+        self.body = self.TexTemplateFileBodyFromFontConfig(self.font)
+
+    def TexTemplateFileBodyFromFontConfig(self, font_name):
```

Comment:
Why is this method named like this? How about `file_body_from_font_config` or something similar?

Reply:
Yeah, agreed, but in my defence, I think the original naming in the Tex Template rewrite was already confusing. TexTemplate derives from TexTemplateFromFile and not the other way around. 
I'll do it better next time.

Reply:
`TexTemplate` derives from `TexTemplateFromFile` because it offers more functionality. The latter is just a wrapper for the old behaviour whereas the former can add packages on-the-fly.

Reply:
I understand the functionality, but I TexTemplateFromFile is a type of TexTemplate, while a TexTemplate generated on the fly by code is not an example of a TexTemplateFromFile.